api/logical.go

```diff
@@ -11,6 +12,26 @@ const (
 	wrappedResponseLocation = "cubbyhole/response"
 )
 
+var (
+	// The default TTL that will be used with `sys/wrapping/wrap`, can be
+	// changed
+	DefaultWrappingTTL = "5m"
+
+	// The default function used if no other function is set, which honors the
+	// env var and wraps `sys/wrapping/wrap`
+	DefaultWrappingLookupFunc = func(operation, path string) string {
+		if os.Getenv(EnvVaultWrapTTL) != "" {
+			return os.Getenv(EnvVaultWrapTTL)
```

Comment:
This is intended, and is the current behavior (if you've globally set an env var,  or done it via `VAULT_WRAP_TTL=5m vault read ...` it should apply to any call you make). If it's explicitly set we want it to override the default val.